source/views/building-hours/lib/get-day-of-week.ts

```diff
@@ -2,4 +2,4 @@ import type {Moment} from 'moment-timezone'
 import type {DayOfWeekEnumType} from '../types'
 
 export const getDayOfWeek = (m: Moment): DayOfWeekEnumType =>
-	m.format('dd') as any as DayOfWeekEnumType
+	m.format('dd') as unknown as DayOfWeekEnumType
```

Comment:
We may be able to get away with this?

```suggestion
	m.format('dd') as DayOfWeekEnumType
```

Reply:
I don't _think_ so, because you can't go straight from "string" to "DayOfWeekEnum"

Reply:
I think this is actually okay to do without type erasure. Typescript seems to be happy with it.

Reply:
I think you should be able to tell Typescript that `string` is `DayOfWeekEnumType` — I don't think that's disallowed:

[Playground](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABHADgQwI4gKYAoCUAXIgM5QBOMYA5ogN4BQizi52UI5SAyhVdQDpg5OAFsAwgAs05cXAAmeAEQABJQIjTZC7AEEouAAz5EAakQBGfAG4GAXwYMoATxTZEAUTAhRAFVfuALyISrpKiAA+IQBCSo4ANuyIAEbEXj7+bojBqJg4BIhoJJ7efgG2DBAIJHCJAvFw1LjJNkA).

For strict enum types, I do think it's restricted.